projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -0,0 +1,44 @@
+{
+    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.bacon": "培根",
+    "item.foodexpansion.cooked_bacon": "香脆培根",
+    "item.foodexpansion.fried_egg": "煎蛋",
+    "item.foodexpansion.bacon_and_egg": "培根煎蛋",
+    "item.foodexpansion.carrot_seed_soup": "胡萝卜汤",
+    "item.foodexpansion.squid": "生鱿鱼",
+    "item.foodexpansion.cooked_squid": "烤鱿鱼",
+    "item.foodexpansion.compressed_flesh": "腊肉",
+    "item.foodexpansion.chocolate_bar": "巧克力棒",
+    "item.foodexpansion.spider_soup": "蛛眼炖汤",
```

```suggestion
    "item.foodexpansion.spider_soup": "蛛眼浓汤",
```
既然`蝙蝠炖汤`改了，那么上下文应一致
此问题请全文通查，尽量是保持`Soup`在模组内的翻译一致